From 12f1889ab7b17a3407f5ee3607043cad19e66d62 Mon Sep 17 00:00:00 2001
From: Po-Hsu Lin <po-hsu.lin@canonical.com>
Date: Thu, 10 Aug 2017 04:07:59 -0400
Subject: [PATCH] UBUNTU: selftests: just build bpf

Just build bpf test for ubuntu_bpf test suite.

Override TARGETS in the end instead of removing other tests.
So that it can be applied to both Zesty and Artful.

Signed-off-by: Po-Hsu Lin <po-hsu.lin@canonical.com>
---
 tools/testing/selftests/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/testing/selftests/Makefile b/tools/testing/selftests/Makefile
index 831022b..28a7fb4 100644
--- a/tools/testing/selftests/Makefile
+++ b/tools/testing/selftests/Makefile
@@ -34,6 +34,7 @@ TARGETS += user
 TARGETS += vm
 TARGETS += x86
 TARGETS += zram
+TARGETS =  bpf
 #Please keep the TARGETS list alphabetically sorted
 # Run "make quicktest=1 run_tests" or
 # "make quicktest=1 kselftest from top level Makefile
-- 
2.7.4

